src/lib/sys/configuration.cpp

```diff
@@ -31,8 +31,14 @@ bool Configuration::loadAndVerifyNvmConfig(void) {
         if(!_flash_partition.read(CONFIG_START_OFFSET_IN_BYTES, reinterpret_cast<uint8_t *>(_ram_partition), sizeof(ConfigPartition_t), CONFIG_LOAD_TIMEOUT_MS)){
             break;
         }
-        uint32_t computed_crc32 = crc32_ieee(reinterpret_cast<const uint8_t *>(&_ram_partition->header.version), (sizeof(ConfigPartition_t)-sizeof(_ram_partition->header.crc32)));
-        if(_ram_partition->header.crc32 != computed_crc32) {
+        size_t buffer_len = sizeof(ConfigPartition_t);
```

Comment:
We probably want to make this smaller. For any realistic short-term use case 256 bytes is probably more than sufficient.

Reply:
Yeah makes sense to me - I think as long as we add debugging provision for the configuration system to panic loudly and obviously when we're hitting encoding buffer limits we're good. 

I don't have a good sense of the encoding efficiency of cbor - Our configurations are on average a [char[20] : uint32]. I think 256 is fine as long as we can do all of the bridge configs (7) plus probably 8 more in the short term?

Reply:
I'm thinking I'll go ahead and make `asCborMap` more robust, including allocating more and retrying if we get `CborErrorOutOfMemory`. I don't want to put our future selves in pain when some future user adds a ton of configs and can't figure out why their hardware doesn't behave as expected.